packages/bpk-component-link/src/BpkLink.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { ReactNode, MouseEvent, AnchorHTMLAttributes } from 'react';
+import { forwardRef } from 'react';
+
+import { cssModules } from '../../bpk-react-utils';
+
+import themeAttributes, {
+  linkAlternateThemeAttributes,
+} from './themeAttributes';
+
+import STYLES from './BpkLink.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+export interface Props
+  extends Omit<AnchorHTMLAttributes<HTMLAnchorElement>, 'href' | 'className' | 'rel'> {
```

Comment:
Use `Omit` to exclude the properties in the parent interface so that redefine them and make `href` also support `null` without TS error